src/search/containers/AddonPage/index.js

```diff
@@ -13,16 +22,92 @@ class AddonPage extends React.Component {
     slug: PropTypes.string.isRequired,
   }
 
+  dataItems() {
+    const { addon } = this.props;
+    const items = [
+      [addon.type, 'type'],
+      [addon.status, 'status'],
+      [<a href={addon.url} target="_blank">{_('View on site')}</a>, 'url'],
+      [<a href={editUrl(addon.url)} target="_blank">{_('Edit on site')}</a>, 'edit'],
```

Comment:
I think it would be better to rely on the API for this URL by adding something like `addon.developer_url`. This is because when URLs change, the server is in a better position to know about it.